packages/rps/package.json

```diff
@@ -9,11 +9,10 @@
     "firebase/**/grpc": "^1.16.1"
   },
   "scripts": {
-    "build:ci": "run-s clearContracts build:typescript contracts:compile",
+    "build:ci": "run-s build:typescript contracts:compile",
```

Comment:
Do contracts need to be compiled? Contracts are already compiled (on every circle run) in `prepare` job.

Reply:
Good question. On the one hand, it's nice for `build:ci` to do the complete build; the whole _point_ of `build:ci` is to check that the package can be built from start to finish. On the other hand, the current `build:ci` is a bit strange: contracts are deleted, then the typescript is compiled, then contracts are compiled, and webpack is _not_ run.

For now, perhaps I'll remove `contracts:compile` from `build:ci`